src/libaktualizr/config/config.cc

```diff
@@ -275,6 +277,12 @@ void Config::postUpdateValues() {
     }
   }
 
+  if (!uptane.director_server.empty()) {
+    if (provision.ecu_registration_endpoint.empty()) {
+      provision.ecu_registration_endpoint = uptane.director_server + "/ecus";
+    }
```

Comment:
General question (not actually specific to this PR): do we expect to ever have some use custom server URLs that aren't just the server URL from the credentials with static names ('/directory', '/repo', '/ecus', etc.) appended?

Reply:
I don't think so. We're relying on mutual TLS for authentication and it is much easier to do it if we always connect to the same server.
It might happen though that ota-commuinity-edition get real users and aktualizr will want to support multiple repo/director deployments with slightly different paths.